src/client/java/teammates/client/scripts/sql/VerifyCourseEntityAttributes.java

```diff
@@ -150,9 +157,87 @@ private boolean verifyStudent(CourseStudent oldStudent,
                 && newStudent.getUpdatedAt().equals(oldStudent.getUpdatedAt())
                 && newStudent.getCreatedAt().equals(oldStudent.getCreatedAt())
                 && newStudent.getRegKey().equals(oldStudent.getRegistrationKey());
+    }
+
+    // Verify Instructor ----------------------------
+    private boolean verifyInstructors(teammates.storage.sqlentity.Course newCourse) {
+        List<teammates.storage.sqlentity.Instructor> newInstructors = getNewInstructors(newCourse.getId());
+        List<Instructor> oldInstructors = ofy().load().type(Instructor.class).filter("courseId", newCourse.getId())
+                .list();
+
+        if (oldInstructors.size() != newInstructors.size()) {
+            return false;
+        }
+
+        newInstructors.sort((a, b) -> a.getEmail().compareTo(b.getEmail()));
+        oldInstructors.sort((a, b) -> a.getEmail().compareTo(b.getEmail()));
+        for (int i = 0; i < oldInstructors.size(); i++) {
+            Instructor oldInstructor = oldInstructors.get(i);
+            teammates.storage.sqlentity.Instructor newInstructor = newInstructors.get(i);
+            if (!verifyInstructor(oldInstructor, newInstructor)) {
+                return false;
+            }
+        }
+        return true;
+    }
+
+    private boolean verifyInstructor(Instructor oldInstructor,
+            teammates.storage.sqlentity.Instructor newInstructor) {
+        InstructorPrivileges oldPrivileges;
+        if (oldInstructor.getInstructorPrivilegesAsText() == null) {
+            oldPrivileges = new InstructorPrivileges(oldInstructor.getRole());
+        } else {
+            InstructorPrivilegesLegacy privilegesLegacy = JsonUtils
+                    .fromJson(oldInstructor.getInstructorPrivilegesAsText(), InstructorPrivilegesLegacy.class);
+            oldPrivileges = new InstructorPrivileges(privilegesLegacy);
+        }
+
+        return newInstructor.getName().equals(oldInstructor.getName())
+                && newInstructor.getEmail().equals(oldInstructor.getEmail())
+                && newInstructor.getRole().getRoleName().equals(oldInstructor.getRole())
+                && newInstructor.getRegKey().equals(oldInstructor.getRegistrationKey())
+                && newInstructor.getDisplayName().equals(oldInstructor.getDisplayedName())
```

Comment:
I think some important attributes to compare are missing ? e.g course, account, team(probably not important for instructor)

Reply:
I verify account in a seperate method, in the #13073. As for course, I thought it was inherently verified since we used courseId to obtain the old and new instructors to compare